tr.org.pardus.pen.yaml

```diff
@@ -0,0 +1,23 @@
+app-id: tr.org.pardus.pen
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+command: /app/usr/bin/pardus-pen
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=host
+modules:
+  - name: pardus-pen
+    buildsystem: simple
+    build-commands:
+      - qmake
+      - make
+      - make install DESTDIR=${FLATPAK_DEST} INSTALL_ROOT=${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://github.com/pardus/pardus-pen/archive/refs/tags/debian/2.0.0.tar.gz
```

Comment:
I don't see a metainfo file being installed or being present in the repo https://github.com/pardus/pardus-pen/


See https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/